### PR TITLE
Allow passing marker options when calling startMarker

### PR DESCRIPTION
**Pull request by dalie**
For example, if you want a marker with a custom icon you can do this `map.editTools.startMarker(null, {icon: myIcon})`.

### CODE REVIEW

**Comment by yohanboniface** [comment]
Nice suggestion :)

What would be nice is:
- make it consistent, so add the same options to `createPolyline`, `createPolygon` and so on
- document it on the API description in the README
- add very simple unittests to enforce that options are passed to the created marker, polyline… instances,  so we don't break it on further devs.

Anyway, thanks for your work! :)

File: src/Leaflet.Editable.js [hunk 1]
```diff
@@ -189,8 +189,9 @@ L.Editable = L.Class.extend({
         return polygon;
     },
 
-    createMarker: function (latlng) {
-        var marker = new this.options.markerClass(latlng, {editOptions: {editTools: this}});
+    createMarker: function (latlng, markerOptions) {
+        var options = L.Util.extend({editOptions: {editTools: this}}, markerOptions);
```
Comment (yohanboniface) [inline src/Leaflet.Editable.js]: No need to create a new var here, let's use `options` as parameter and do

`options = L.Util.extend…`

**Comment by dalie** [comment]
One unrelated test fails, L.PolygonEditor #dragMiddleMarker(). Not sure if it's something I did or something else. Any help would be appreciated.

**Comment by yohanboniface** [comment]
Humm, try upgrading leaflet to latest master "Leaflet/Leaflet" instead of "1.0.0-beta1".

**Comment by dalie** [comment]
Turns out I had to remove the layer I created in the test.

**Comment by yohanboniface** [comment]
Thanks, great contrib :)

**Comment by yohanboniface** [comment]
Can you issue the same PR against master?

**Pull request merged**